### PR TITLE
docs: using Dyno

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

It adds a section to explain how to use Dyno with a remote APM Server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+  --with-opbeans-python \
+  --opbeans-python-service-environment local \
+  --opbeans-node-service-version test-demo \
+  --opbeans-python-service-name dyno-service
+```
+
+when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
+in this case we have disabled PostgreSQL to cause a database service error.
+
+![](images/dyno.png)
```
Comment (kuisathaverat) [inline README.md:247]:
```suggestion
![](images/dyno.png|width=400)
```

File: README.md [hunk 2]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+
+Then we can check the result of our changes in Dyno in the APM UI, in this case we can see that
+the error rate for postgreSQL is increasing.
+![](images/apm-dyno.png)
```
Comment (kuisathaverat) [inline README.md:251]:
```suggestion
![](images/apm-dyno.png|width=400)
```

File: README.md [hunk 3]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
 |Node|:black_square_button:|
 |Ruby|:black_square_button:|
 
+### Using Dyno with a remote APM Server
+
+It is possible to connect the infrastructure generated by compose.py to a remote APM Server.
+This makes possible to send APM date from the scenarios modelled with Dyno.
```
Comment (cachedout) [inline README.md:224]:
```suggestion
This makes possible to send APM data from the scenarios modeled with Dyno.
```

**Review by cachedout — APPROVED** [review]
💯 Nice!

File: README.md [hunk 4]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+  --with-opbeans-python \
+  --opbeans-python-service-environment local \
+  --opbeans-node-service-version test-demo \
+  --opbeans-python-service-name dyno-service
+```
+
+when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
+in this case we have disabled PostgreSQL to cause a database service error.
+
+![](images/dyno.png|width=400)
```
Comment (kuisathaverat) [inline README.md:247]:
```suggestion
![](images/dyno.png|width=400px)
```

**Review by mdelapenya — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (kuisathaverat) [inline README.md]:
```suggestion
![](images/dyno.png | width=400)
```

File: README.md [hunk 5]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+  --with-opbeans-python \
+  --opbeans-python-service-environment local \
+  --opbeans-node-service-version test-demo \
+  --opbeans-python-service-name dyno-service
+```
+
+when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
+in this case we have disabled PostgreSQL to cause a database service error.
+
+![](images/dyno.png|width=400px)
```
Comment (kuisathaverat) [inline README.md:247]:
```suggestion
![](images/dyno.png | width=400)
```

**Review by v1v — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+```
+
+when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
+in this case we have disabled PostgreSQL to cause a database service error.
+
+![](images/dyno.png | width=400)
+
+Then we can check the result of our changes in Dyno in the APM UI, in this case we can see that
+the error rate for postgreSQL is increasing.
+![](images/apm-dyno.png|width=400)
```
Comment (kuisathaverat) [inline README.md:251]:
```suggestion
![](images/apm-dyno.png | width=400)
```

File: README.md [hunk 7]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+  --with-opbeans-python \
+  --opbeans-python-service-environment local \
+  --opbeans-node-service-version test-demo \
+  --opbeans-python-service-name dyno-service
+```
+
+when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
+in this case we have disabled PostgreSQL to cause a database service error.
+
+![](images/dyno.png | width=400)
```
Comment (kuisathaverat) [inline README.md:247]:
```suggestion
![](images/dyno.png)
```

File: README.md [hunk 8]
```diff
@@ -218,6 +218,38 @@ Not all Opbeans are supported for use with Dyno
...
+
+Then we can check the result of our changes in Dyno in the APM UI, in this case we can see that
+the error rate for postgreSQL is increasing.
+![](images/apm-dyno.png | width=400)
```
Comment (kuisathaverat) [inline README.md:251]:
```suggestion
![](images/apm-dyno.png)
```

**Pull request merged**